modules/t/archiveStableId.t

```diff
@@ -227,6 +227,13 @@ $archive_obj = $asia->fetch_by_stable_id('ENSG00000171456.1');
 is($archive_obj->stable_id, 'ENSG00000171456', 'fetch_by_stable_id with active stable_id with version');
 ok($archive_obj->is_latest, 'Is the latest stable_id');
 
+# Test for a possible mock of MultiSpecies entry defined in species_id field form the MappingSession table. 
+# Modified SQL statement in function _fetch_archive_id() from file ArchiveStableIdAdaptor.pm
+
+$archive_obj = $asia->fetch_by_stable_id("T7");
+is($archive_obj->stable_id, 'T7', 'fetch_by_stable_id with active stable_id found for a new species');
+is($archive_obj->species_id, 2, 'expected species_id == 2 is true');
```

Comment:
cmp_ok() is a better comparator for numeric values. is() only compares strings. It works here anyway because Perl stringifies the integer for comparison.